lib/Adapters/AdapterFactory.js

```diff
@@ -2,15 +2,22 @@
 var WowzaAdapter=require('./WowzaAdapter.js');
 var APIQueryAdapter=require('./APIQueryAdapter.js');
 var TestAdapter=require('./TestAdapter.js');
-var RegressionAdapter=require('./RegressionAdapter.js');
-
+var RegressionAdapter=require('./RegressionAdapter/RegressionAdapter.js');
+var process = require('process');
 var config = require('../../common/Configuration');
 
 var simulateStreams = config.get('simulateStreams');
 var regressionAdapterConfig = config.get('regressionAdapter');
+var command = process.argv.length > 2 ? process.argv[2] : undefined;
+
 
 exports.getAdapter = function() {
 
+    if (command != undefined) {
```

Comment:
this doesn't belong here  if you want to override configuration it should be in a code that handles config (maybe config.js...)